src/SquirrelInstaller/Bloom.nuspec

```diff
@@ -13,7 +13,10 @@
 	<iconUrl>https://s3.amazonaws.com/bloomlibrary.org/squirrel/BloomSetup.ico</iconUrl>
   </metadata>
 	<files>
-		<file src="..\..\output\release\**\*.*" exclude="*.xml;Bloom.vshost.exe;AWSSDK.pdb;remoteDebugging" target="lib\net45\"/>
+		<file src="..\..\output\release\**\*.*" exclude="*.xml;Bloom.vshost.exe;**\*.pdb;**\BloomTests.*;remoteDebugging" target="lib\net45\"/>
+		<!-- Having excluded all pdbs, we do want the main one for the Bloom exe, so we get better stack dumps.
+		This pattern should cover Bloom.pdb or BloomAlpha.pdb or anything similar (but we don't need BloomTests.pdb or BloomChorusPlugin.pdb).-->
```

Comment:
So, should we make a new card to add them in? About 3M uncompressed, probably about 0.6 compressed.